crates/swc_common/src/syntax_pos.rs

```diff
@@ -199,6 +200,10 @@ pub struct MultiSpan {
     span_labels: Vec<(Span, String)>,
 }
 
+extern "C" {
```

Comment:
I think we need `#[cfg]`

Reply:
isn't this guarded via https://github.com/swc-project/swc/pull/4268/files#diff-f2c8fa127c4fc658c5b278fa9d6084d2b8c45070d1fc7515191d2afa981bbd8bR446?

this is same as existing, i.e https://github.com/swc-project/swc/blob/main/crates/swc_common/src/syntax_pos/hygiene.rs#L78=

Reply:
Ah, I see, thanks!